Fix edit URL in new comment e-mail

RavuAlHemio opened the pull request:
The primary backend endpoint has moved from `BASEURL/contao/main.php` to `BASEURL/contao`; this PR updates the URL sent in the "a new comment has been created" e-mail accordingly.

aschempp commented:
Thanks! I think the correct approach in Contao 4/ Symfony would be to use the router instead of again hard coding the URL. Are you familiar with that to update the PR?

RavuAlHemio committed:
Fix edit URL in new comment e-mail.

The comments backend module has moved from contao/main.php?do=comments to
contao?do=comments.

RavuAlHemio commented:
I'm a bit inexperienced, but how does this look?

leofeyer commented:
Fixed in f728fcddd1c80c38f83200ad962b095b0bdfcec7. Once we switch to using the router everywhere in the back end, we can implement a more sophisticated solution :)
